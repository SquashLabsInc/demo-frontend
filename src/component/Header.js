import React, { Component } from 'react';
import { Nav, Navbar } from "react-bootstrap";
  
class Header extends Component {
    render() {
        return (
            <React.Fragment>
                <Navbar bg="dark">
                    <Navbar.Brand href="https://www.squash.io/"><img src="https://www.squash.io/wp-content/uploads/2019/03/logo.svg" alt="" width="157" height="22" title="Squash" className="blog-hide"></img></Navbar.Brand>
                    <Navbar.Toggle />
                    <Navbar.Collapse className="justify-content-end">
                        <Nav defaultActiveKey="/" as="ul">
                            <Nav.Item as="li">
                                <Nav.Link href="/">Home</Nav.Link>
                            </Nav.Item>
                        </Nav>
                    </Navbar.Collapse>
                </Navbar>
            </React.Fragment>
        )
    }
}

export default Header

