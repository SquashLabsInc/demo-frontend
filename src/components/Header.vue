<template>
    <div>
        <b-navbar toggleable="lg" type="dark" variant="info">
            <b-navbar-brand href="#"><img src="https://www.squash.io/wp-content/uploads/2019/03/logo.svg" alt="" width="157" height="22" title="Squash" class="blog-hide"/></b-navbar-brand>

            <b-navbar-toggle target="nav-collapse"></b-navbar-toggle>

            <b-collapse id="nav-collapse" is-nav>
            <b-navbar-nav>
                <b-nav-item href="#">Document</b-nav-item>
            </b-navbar-nav>
            </b-collapse>
        </b-navbar>
    </div>
</template>

<script>
export default {
  name: 'Header',
  props: {
    msg: String
  }
}
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped>
h3 {
  margin: 40px 0 0;
}
ul {
  list-style-type: none;
  padding: 0;
}
li {
  display: inline-block;
  margin: 0 10px;
}
a {
  color: #42b983;
}
</style>
