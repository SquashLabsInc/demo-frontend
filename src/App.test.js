import React from 'react';
import { render } from '@testing-library/react';
import { create } from "react-test-renderer";
import Header from "./component/Header"

describe("Button component", () => {
  test("Matches the snapshot", () => {
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
  });
});