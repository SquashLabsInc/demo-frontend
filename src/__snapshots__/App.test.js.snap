// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
<nav
  className="navbar navbar-expand navbar-light bg-dark"
>
  <a
    className="navbar-brand"
    href="https://www.squash.io/"
  >
    <img
      alt=""
      className="blog-hide"
      height="22"
      src="https://www.squash.io/wp-content/uploads/2019/03/logo.svg"
      title="Squash"
      width="157"
    />
  </a>
  <button
    aria-label="Toggle navigation"
    className="navbar-toggler collapsed"
    onClick={[Function]}
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
  <div
    aria-expanded={null}
    className="justify-content-end navbar-collapse collapse"
  >
    <ul
      className="navbar-nav"
      onKeyDown={[Function]}
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link active"
          data-rb-event-key="/"
          href="/"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          Home
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
